src/docker-utils.coffee

```diff
@@ -204,8 +204,7 @@ do ->
 						docker.getImage(tag).removeAsync()
 						.then ->
 							console.log('Deleted image:', tag, image.Id, image.RepoTags)
-						.catch (err) ->
-							console.log('Error deleting image:', tag, image.Id, image.RepoTags, err)
+						.catch (err) -> # Ignore image delete error
```

Comment:
No need for `err` here.
Also might look nicer by defining:
```
ignore  = ->
```
somewhere above,and then here (and in the other error to ignore) do this:
```
.catch(ignore)
```